app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command_split = command.split(" ")
```

Comment:
you need to make sure that the command is "cp" and that the length of command_split is 3